set.mm

```diff
@@ -33322,20 +33318,27 @@ general is seen either by substitution (when the variable ` V ` has no
   $}
 
   ${
-    $d x y $.  $d x A $.  $d x y B $.  $d x y ps $.
+    $d x A $.  $d x y B $.  $d x y ps $.
     vtocl2.1 $e |- A e. _V $.
     vtocl2.2 $e |- B e. _V $.
     vtocl2.3 $e |- ( ( x = A /\ y = B ) -> ( ph <-> ps ) ) $.
     vtocl2.4 $e |- ph $.
     $( Implicit substitution of classes for setvar variables.  (Contributed by
-       NM, 26-Jul-1995.)  (Proof shortened by Andrew Salmon, 8-Jun-2011.) $)
+       NM, 26-Jul-1995.)  (Proof shortened by Andrew Salmon, 8-Jun-2011.)
+       (Proof shortened by Wolf Lammen, 10-Apr-2025.) $)
     vtocl2 $p |- ps $=
+      ( wtru cvv wcel a1i vtocl2d mptru ) BKABCDEFLLELMKGNFLMKHNIAKJNOP $.
+
+    $( Implicit substitution of classes for setvar variables.  (Contributed by
```

Comment:
Write the date of revision: `Obsolete version of ~ vtocl2 as of 10-Apr-2025.` so that it will be known when to delete it.

Reply:
Fixed